Integrations/integration-FireEyeHX.yml

```diff
@@ -2379,7 +2460,8 @@ script:
             start_search()
         elif command == 'fireeye-hx-get-host-set-information':
             get_host_set_information()
-        sys.exit(0)
+        elif command == 'fireeye-hx-bulk-append-conditions-request':
+            bulk_append_conditions_request()
```

Comment:
you should split the function into two: command and request, and you should call the command function here and not the request